### ISSUE TITLE
Asteroids attached to ship

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#169]
Asteroids (and other objects) get attached to your ship and appear to float with you during cutscenes and while in ship-to-ship combat.

---

## Details 
Asteroids (and other objects) get attached to your ship.  They can be identified with console commands and removed that way, but this will likely affect achievements.

## Game Version
E235D2B4 v1.7.23.0

## New Game
No

## Reproduction Steps
Unable to recreate consistently, though I have noted that this has been been reported by the community (reddit, discord, forums), so it could be a widespread issue.

**Comment by Constellation-VASCO** [comment]
One of my characters has 3 asteroids following his ship around like this describes. Due to another serious main quest issue I halted playing this character after about 70 hours and started a new one. The new one does not have any 'attached asteroids' yet after about 20 hours of playtime, so it's definitely intermittent.

**Comment by Constellation-VASCO** [comment]
I have the same issue. I collected 6 or 7 asteroids by now. Still playable but a bit annoying.

I noticed them for sure when there were 3 of them. Since then I tried to observe when they get added. And it seems to happen when I blow up asteroids in the orbit of planets to harvest resources. I'm pretty sure of this because the count was stable at 4 for a long time until I resumed blowing up asteroids.

**Comment by Constellation-VASCO** [comment]
Give this new mod fix a go -

https://www.nexusmods.com/starfield/mods/5975/

**Comment by Constellation-VASCO** [comment]
I actually found the issue consistent for me after finding that I was doing heavy Modding for ship storage values.
Often times I am finding this more and more common after looting ships or resources in the asteroid fields, even found those asteroid assets followed my ship down to the ground.
So I think the Mod that was listed by Honem should be included in the Community Patch Mod as well for the fix .... after first glance ... haven't had time to test it yet.

**Comment by Constellation-VASCO** [comment]
wSkeever figured it out.

For some reason the game is placing both your starship and the asteroids in a special galaxy cell. This is why they follow you around , they're sort of attached to your ship in a way.

What their mod does is start a quest that automatically finds 10 object references in that galaxy cell that's NOT a ship and deletes them each time your travel and/or load game (travel as it fast travel or grav jump to different system). 

This means for those poor souls that have entire star stations or even New Atlantis in one case ... the mod may work for them as well as it sounds like they're also getting objects stuck in their galaxy cell.

**Comment by Constellation-VASCO** [comment]
oh and if you have more then 10 ? Keep grav jumping until all gone I guess :)

And the mod is cleaner then finding the asteriod's object ref via console (it'll be a CONT ref) and using the disable command. Disable only hides it , it doesn't delete it.

**Comment by Constellation-VASCO** [comment]
so then it would be more ideal for them to implement that mod feature into the patch?
Because I will be honest I am seeing alot of Check value error's for alot of my bugs.
And one of them was for ship error's including unique ship object items staying locked in place and registering disconnected and floating in a similar state as those asteroids.

**Comment by Constellation-VASCO** [comment]
Yes it would be more ideal for them to implement in the community patch and in fact the author of the mod explicitly gives permission to the Community Patch team to include it.

**Comment by Constellation-VASCO** [comment]
@honem just some feedback that https://www.nexusmods.com/starfield/mods/5975/ does not work for me. I finally got SFSE, Plugins.txt Enabler and the fix installed properly. Found out that it's loaded by getting an "Achievements disabled due to active mods" notice when continuing after starting Starfield. However, neither saving and reloading nor grav-jumping several times removed any of my (by now) 9+ asteroids.
